widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.kt

```diff
@@ -807,20 +812,17 @@ internal class ChatView(context: Context, attrs: AttributeSet?, defStyleAttr: In
         binding.scBottomBannerLabel.applyTextTheme(secureMessagingTheme.bottomBannerText)
         binding.scBottomBannerLabel.applyLayerTheme(secureMessagingTheme.bottomBannerBackground)
         binding.scBottomBannerDivider.applyColorTheme(secureMessagingTheme.bottomBannerDividerColor)
+        binding.scErrorLabel.applyLayerTheme(secureMessagingTheme.unavailableStatusBackground)
+        binding.scErrorLabel.applyTextTheme(secureMessagingTheme.unavailableStatusText)
+        secureMessagingTheme.unavailableStatusText
+            ?.textColor
+            ?.primaryColorStateList
+            ?.let(binding.scErrorLabel::setCompoundDrawableTintList)
```

Comment:
I was not sure about this solution and wanted to highlight it for reviewers.
- This changes the color of `app:drawableStartCompat="@drawable/ic_attention"` that is part of TextView properties.
- I just applied the same color as text to it, but should it have a separate property in remote Json schema?
- This solution does not support gradient colors even if text does
- Should I move this solution as part of the [applyTextTheme](https://github.com/salemove/android-sdk-widgets/blob/3c499a6624bc787a276956bca46d07aa514d3625/widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/UnifiedUiExtensions.kt#L134)?

Reply:
cc: @igorkravchenko

Reply:
> I just applied the same color as text to it, but should it have a separate property in remote Json schema?

I think it should not have a separate color configuration. because it visually looks like a part of the text.

> Should I move this solution as part of the [applyTextTheme](https://github.com/salemove/android-sdk-widgets/blob/3c499a6624bc787a276956bca46d07aa514d3625/widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/UnifiedUiExtensions.kt#L134)?

I like the idea.

Reply:
> * I just applied the same color as text to it, but should it have a separate property in remote Json schema?
@gugalo I did the same for iOS, with the reasoning of Davit.